src/components/LooMap/LocateMapControl.js

```diff
@@ -58,11 +58,20 @@ const LocateMapControl = ({ position }) => {
     onStopLocation,
   });
 
+  const handleClick = () => {
+    if (!isActive) {
```

Comment:
not a huge deal really, but swapping the condition so we're not using the not operator (like the condition was before the change) might help with readability